catalog/core/catalog-core-solrcommands/pom.xml

```diff
@@ -25,6 +25,21 @@
     <name>DDF :: Catalog :: Core :: Solr Commands</name>
     <packaging>bundle</packaging>
     <dependencies>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+            <version>4.5.3</version>
```

Comment:
❓ Looks like these used to be managed dependencies. They're embedded, but do we want to get back to that at some point?

Reply:
After the Solr as a separate process PR is merged, we should be able to go back and create a lightweight Solrj jar/bundle to use instead.